morpheus/utils/downloader.py

```diff
@@ -62,12 +62,13 @@ class Downloader:
         The heartbeat interval to use when using dask or dask_thread.
     """
 
+    _dask_cluster = None
```

Comment:
Important: Add a comment around this describing that this is a cluster shared by all uses of `Downloader` instances.